docs/index.rst

```diff
@@ -133,7 +133,21 @@ The following example initializes the extension with the standard Flask command-
     from flask_migrate import Migrate
     migrate = Migrate(app, db)
 
-The two arguments to ``Migrate`` are the application instance and the Flask-SQLAlchemy database instance. The ``Migrate`` constructor also takes additional keyword arguments, which are passed to Alembic's ``EnvironmentContext.configure()`` method. As is standard for all Flask extensions, Flask-Migrate can be initialized using the ``init_app`` method as well.
+The two arguments to ``Migrate`` are the application instance and the Flask-SQLAlchemy database instance. The ``Migrate`` constructor also takes additional keyword arguments, which are passed to Alembic's ``EnvironmentContext.configure()`` method. As is standard for all Flask extensions, Flask-Migrate can be initialized using the ``init_app`` method as well::
+
+    from flask_migrate import Migrate
+
+    migrate = Migrate()
+
+    def create_app():
+         ...
+         # instantiate your db
+         ...
+         migrate.init_app(app, db)
+         ...
+         # import your models
```

Comment:
The models do not necessarily need to be imported here, and a lot of people prefer to not do imports inside functions, so I would remove this comment and leave it up to the developer to decide where to do the import.